.gitignore

```diff
@@ -2,3 +2,4 @@
 **/*.pyc
 **/.DS_Store
 .coverage
+**/bot_token.txt
```

Comment:
Aggiungi `**/*.png`, in modo tale che vengano ignorati anche i file immagine dei codici qr generati

Reply:
fatto